src/main/java/com/minecolonies/coremod/commands/generalcommands/CheckForAutoDeletesCommand.java

```diff
@@ -76,8 +76,8 @@ else if (sender instanceof TileEntity)
             {
                 sender.sendMessage(new TextComponentString("Successful"));
                 for (final Colony col : coloniesToDelete)
-                {
-                    server.addScheduledTask(() -> ColonyManager.deleteColony(col.getID()));
+                {//TODO: bugcheck
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3ATrailingCommentCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1135)